experimental/logging/log_listener.go

```diff
@@ -41,6 +41,10 @@ type loggingListenerFactory struct {
 // NewListener implements the same method as documented on
 // experimental.FunctionListener.
 func (f *loggingListenerFactory) NewListener(fnd api.FunctionDefinition) experimental.FunctionListener {
+	if fnd.ModuleName() == "internal/testing/proxy/proxy.go" {
```

Comment:
how about having this under testing/loggingListener and override NewListener rather than having this "test code" in the main code

Reply:
better idea indeed. lemme try!

Reply:
maybe under testing/proxy/ would be just good